README.md

```diff
@@ -1,26 +1,85 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+let num = 20
+while (num >= 0) {
+  console.log(num)
+  num --
+}
+
+for (num = 20; num >= 1; num -= 1){
+  console.log (num)
+}
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+
+let num = 20
+while (num <=20 && num >= 0) {
+if (num === 0){
+  console.log (num +' is zero')
+  }
+  if (num %2 === 0 && num !== 0){
```

Comment:
You don't have to add the second bit of logic here - the number is already not zero because we captured zero in the first bit of logic